content/blog/2021/06/2021-06-17-libera-chat.adoc

```diff
@@ -0,0 +1,20 @@
+---
+:layout: post
+:title: "Jenkins IRC on Libera Chat"
+:tags:
+- community
+:author: markewaite
+opengraph:
```

Comment:
Maybe you want to pre-register a Discourse conversation so that it is present once #4424 is integrated

Reply:
Attempted in ee97a9e316881f2bd09b3631f2876f752362655f